types/prosemirror-transform/index.d.ts

```diff
@@ -7,17 +7,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import {
-  ContentMatch,
-  Fragment,
-  Mark,
-  MarkType,
-  Node as ProsemirrorNode,
-  NodeRange,
-  NodeType,
-  Schema,
-  Slice
-} from 'prosemirror-model';
+import { ContentMatch, Fragment, Mark, MarkType, Node as ProsemirrorNode, NodeRange, NodeType, Schema, Slice } from 'prosemirror-model';
```

Comment:
Nothing really changed here, right?

(May your Prettier config did change. :-D )

Reply:
You're right. My editor applied some formatting changes. I fixed the issue and reverted those unnecessary changes.